src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -512,7 +512,7 @@ internal sealed override TypeSymbolWithAnnotations GetFieldType(ConsList<FieldSy
             }
 
             // update the lazyType only if it contains value last seen by the current thread:
-            if ((object)Interlocked.CompareExchange(ref _lazyType, type.WithModifiers(this.RequiredCustomModifiers), null) == null)
+            if ((object)Interlocked.CompareExchange(ref _lazyType, type.WithModifiers(this.RequiredCustomModifiers).SetUnknownNullabilityForReferenceTypesIfNecessary(ContainingModule), null) == null)
```

Comment:
>SetUnknownNullabilityForReferenceTypesIfNecessary [](start = 116, length = 49)

I didn't understand why `SetUnknownNullabilityForReferenceTypesIfNecessary` is used here. I'll stop by.

Reply:
`type` is set through several code paths above so it was simplest to erase nullability in one place. The comment for `SetUnknownNullabilityForReferenceTypesIfNecessary` in `PEParameterSymbol` should cover this and other calls to this method.

---
In reply to: [171079071](https://github.com/dotnet/roslyn/pull/25087#discussion_r171079071) [](ancestors = 171079071)